test/tests/rackhd20/test_rackhd20_api_ucs.py

```diff
@@ -0,0 +1,297 @@
+'''
+Copyright 2017, EMC, Inc.
+
+Author(s):
+
+UCS test script that tests:
+-All the ucs service APIs
+-The Discovery workflow
+-The Catalog workflow
+
+'''
+
+import fit_common
+import time
+from nosedep import depends
+import flogging
+from nose.plugins.attrib import attr
+
+logs = flogging.get_loggers()
+
+
+@attr(all=True, regression=True, smoke=True)
+class rackhd20_api_nodes(fit_common.unittest.TestCase):
```

Comment:
Also, change the class name, this shows up as the test suite.
ucs_api_node  or rackhd_ucs_api_tests...something good.
remove 'fit_common.', not needed  (when you update the imports).
  ie.  just need (unittest.TestCase)